examples/all-clusters-app/all-clusters-common/gen/endpoint_config.h

```diff
@@ -344,7 +344,7 @@
     }
 
 // Largest attribute size is needed for various buffers
-#define ATTRIBUTE_LARGEST (254)
+#define ATTRIBUTE_LARGEST (255)
```

Comment:
these do not seem to be pull-modules changes ... shouold we have this delta?

Reply:
I re-ran the ZAP generation and this came out. Honestly I have no idea why.

Reply:
@vivien-apple Do you have any idea why?

Reply:
To answer my own question, from the zap repo:
```
commit f74ebf959d86769ae72ca4ff60e02c5b0bcd4ea9
...
    - Incrementing the attribute largest return by 1 since that is how our embedded code assumes it
```

But does _CHIP_ code assume it?  @tecimovic @brdandu